voronoi_game.py

```diff
@@ -125,18 +170,48 @@ def __init__(self, player_list, args):
         print("Day Score - {}".format(result["player_score"]))
         print("Total Score - {}".format(result["player_total_score"]))
         print("{} Units - {}".format(result["player_names"][0], result["unit_id"][0]))
-        print("{} Unit Positions - {}".format(result["player_names"][0],
-                                              [(float(i.x), float(i.y)) for i in result["unit_pos"][0]]))
+        print(
+            "{} Unit Positions - {}".format(
+                result["player_names"][0],
+                [(float(i.x), float(i.y)) for i in result["unit_pos"][0]],
+            )
+        )
         print("{} Units - {}".format(result["player_names"][1], result["unit_id"][1]))
-        print("{} Unit Positions - {}".format(result["player_names"][1],
-                                              [(float(i.x), float(i.y)) for i in result["unit_pos"][1]]))
+        print(
+            "{} Unit Positions - {}".format(
+                result["player_names"][1],
+                [(float(i.x), float(i.y)) for i in result["unit_pos"][1]],
+            )
+        )
         print("{} Units - {}".format(result["player_names"][2], result["unit_id"][2]))
-        print("{} Unit Positions - {}".format(result["player_names"][2],
-                                              [(float(i.x), float(i.y)) for i in result["unit_pos"][2]]))
+        print(
+            "{} Unit Positions - {}".format(
+                result["player_names"][2],
+                [(float(i.x), float(i.y)) for i in result["unit_pos"][2]],
+            )
+        )
         print("{} Units - {}".format(result["player_names"][3], result["unit_id"][3]))
-        print("{} Unit Positions - {}".format(result["player_names"][3],
-                                              [(float(i.x), float(i.y)) for i in result["unit_pos"][3]]))
-        print("\nTime Elapsed - {:.3f}s".format(self.end_time-self.start_time))
+        print(
+            "{} Unit Positions - {}".format(
+                result["player_names"][3],
+                [(float(i.x), float(i.y)) for i in result["unit_pos"][3]],
+            )
+        )
+        print("\nTime Elapsed - {:.3f}s".format(self.end_time - self.start_time))
+        if args.dump_state:
```

Comment:
Is there any other logical change to `voronoi_game.py` apart from these lines?

A large portion of the PR is just formatting changes, making it hard to identify actual logic changes.

Reply:
Correct that this was the only change. Reverted formatting changes. Thanks!